web/app.js

```diff
@@ -737,6 +738,12 @@ let PDFViewerApplication = {
       }
     }
 
+    if (this.url && isFileSchema(this.url)) {
+      let appConfig = PDFViewerApplication.appConfig;
```

Comment:
Why `PDFViewerApplication.appConfig` here? You could, and even should, simply use `this.appConfig` here.